resources/public/css/orchestrator.css

```
@@ -55,25 +57,236 @@ body {
+.hidden { display: none !important; }
+.invisible { visibility: hidden !important; }
+
+.sr-only,
+.sr-only-focusable:not(:focus):not(:focus-within) {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)